pkg/virt-controller/watch/vm.go

```diff
@@ -691,7 +691,9 @@ func (c *VMController) setupVMIFromVM(vm *virtv1.VirtualMachine) *virtv1.Virtual
 
 	setupStableFirmwareUUID(vm, vmi)
 
-	vmi.ObjectMeta.Annotations = map[string]string{}
+	if vmi.ObjectMeta.Annotations == nil {
```

Comment:
Copying annotation is now irelevant here, since you nicely introduced into the template.